library/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
     compile('com.android.support.test.espresso:espresso-contrib:2.0') {
         exclude module: 'support-v4'
     }
+    compile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
+    compile 'com.android.support:appcompat-v7:25.1.0'
```

Comment:
Yes, it's a must. It's related with this line of PermissionGranter:
`int permissionStatus = ContextCompat.checkSelfPermission(context, permissionNeeded);`

By the way... we should change the exclude that we recommend in the readme, 'cause if excluding this dependancy, PermissionGranter will fail...